### DISCUSSION TITLE
Use all remainding storage when stoping nextcloud from AIO

### DISCUSSION THREAD

**Discussion by Boubik** [nextcloud/all-in-one#3638] [Questions] [answered]
### Steps to Reproduce
1. Click on "Stop" in Nextcloud AIO.
2. Run `df -Th`.

### Expected Behavior 
It should stop in a few minutes and not use all the remaining storage (system storage "/" where data are not stored).

### Actual Behavior 
It takes about 15 minutes or even more to stop, and my system disk has 0 bytes available. Before stopping, there is around 60GB of free space. Also, when running, it doesn't work as it should. I can't log in on new tabs, the Nextcloud app doesn't recognize it, and the web UI where I can access it (only from one PC) looks completely wrong, with no background color, icons all over the place, and more.

### Host OS 
Debian 12

### Nextcloud AIO Version 
Nextcloud AIO v7.5.0

### Current Channel 
Stable

### Other Valuable Information 
It all started with the last update. As usual, I clicked "Stop" and updated containers. It updated to v7.5.0 (from v7.4.1), but the update button was still there. I tried to update it about five times, but the result was the same. The version on top says v7.5.0, but the update button remains at the bottom. So, I left it overnight, and today, I tried the same thing but with an SSH connection. I found that the system disk is full. I deleted some unimportant files and tried to update. Finally, it updated, and the update button disappeared. However, when I'm trying to connect to Nextcloud, it doesn't work as it should. In AIO, it says everything is running. Still, when I try to stop the containers (from AIO), it slowly consumes all my system space, taking more than 15 minutes to stop and leaving me with 0 bytes of free space.

[img](https://pasteboard.co/qWny81y9TEjm.png)

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

**Reply by Boubik** [reply]
Thank you for your assistance. However, now it is showing the following message: "It seems that at least one container is currently restarting, indicating it is unable to start correctly. To escape this endless loop, please stop the containers listed below and examine the container logs before attempting to start them again."

And in log of Nextcloud there is this.
```
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n /dev/dri ']'
++ find /dev/dri -maxdepth 1 -mindepth 1 -name renderD128
+ '[' -n /dev/dri/renderD128 ']'
++ stat -c %g /dev/dri/renderD128

6
+ groupadd -g 106 render2
++ getent group 106
++ cut -d: -f1
+ GROUP=render2
+ usermod -aG render2 www-data
+ touch /dev-dri-group-was-added
+ set +x
Installing imagemagick via apk...
Enabling Imagick...
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/main: No such file or directory
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/community: No such file or directory
Configuring Redis as session handler...
Setting php max children...
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
+ '[' -f /dev-dri-group-was-added ']'
+ set +x
Configuring Redis as session handler...
Setting php max children...
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
+ '[' -f /dev-dri-group-was-added ']'
+ set +x
```
And the issue continues to persist. I checked the link provided in the log, and it mentions a way to reset the installation, but it also mentions that it will delete all my data. That's not the solution I'm looking for. Is there an alternative approach to resolve this issue without losing my data?

Thank you for all your support.

**Reply by szaimen** [reply]
> Is there an alternative approach to resolve this issue without losing my data?

Yes, restore a backup

**Reply by Boubik** [reply]
Okay, that isn't an option for me because I don't have a backup using the tool on AIO. I do have RAID 1 on the data disk. Nonetheless, thank you for all your hard work and assistance. Have a great day or evening!
